cmake/StaticBuild.cmake

```diff
@@ -258,6 +259,10 @@ if(CMAKE_CROSSCOMPILING)
     set(openssl_arch linux-mips64)
   elseif(ARCH_TRIPLET STREQUAL mips-linux-gnu)
     set(openssl_arch linux-mips32)
+  elseif(ARCH_TRIPLET STREQUAL mips-openwrt-linux)
+    # needed to fix unbound so that it links correctly with openssl (crypto, EVP_sha256)
```

Comment:
this comment seems like it'd be best in a commit message :^D

Reply:
okay xD

Reply:
just I leave the comment just in case though?

Reply:
i think having it in the commit only is better so i git blame preserves the info

Reply:
oki